pkg/broker/broker.go

```diff
@@ -718,3 +724,35 @@ func (a AnsibleBroker) AddSpec(spec apb.Spec) (*CatalogResponse, error) {
 	service := SpecToService(&spec)
 	return &CatalogResponse{Services: []Service{service}}, nil
 }
+
+func (a AnsibleBroker) RemoveSpec(specID string) error {
+	spec, err := a.dao.GetSpec(specID)
+	if client.IsKeyNotFound(err) {
+		return ErrorNotFound
+	}
```

Comment:
If someone explicitly requests the removal of a spec and it doesn't exit, I think that's totally a valid error case.

"Please remove spec X", "We have no knowledge at all of spec X, bad request"